caffe2/python/operator_test/channel_shuffle_test.py

```diff
@@ -9,27 +9,40 @@
 from hypothesis import given
 import hypothesis.strategies as st
 
+
 class ChannelShuffleOpsTest(hu.HypothesisTestCase):
     @given(
         channels_per_group=st.integers(min_value=1, max_value=5),
         groups=st.integers(min_value=1, max_value=5),
         n=st.integers(min_value=1, max_value=2),
+        order=st.sampled_from(["NCHW", "NHWC"]),
         **hu.gcs)
-    def test_channel_shuffle(self, channels_per_group, groups, n, gc, dc):
+    def test_channel_shuffle(self, channels_per_group, groups, n, order, gc, dc):
         X = np.random.randn(
             n, channels_per_group * groups, 5, 6).astype(np.float32)
+        if order == "NHWC":
+            # NCHW -> NHWC
+            X = X.transpose((0, 2, 3, 1))
 
         op = core.CreateOperator("ChannelShuffle", ["X"], ["Y"],
-                                 group=groups, kernel=1)
+                                 group=groups, kernel=1, order=order,
+                                 device_option=dc)
```

Comment:
Should be passing gc here (or you can ignore it because assertReferenceChecks will do that for you).